src/controllers/user/me.rs

```diff
@@ -38,10 +38,10 @@ pub fn me(req: &mut dyn Request) -> CargoResult<Response> {
 
     let verified = verified.unwrap_or(false);
     let verification_sent = verified || verification_sent;
-    let user = User { email, ..user };
+    let user = User { ..user };
```

Comment:
I think this line can be completely removed now, actually! I think `user = User { ..user }` is doing a no-op.